app/helpers/application_helper.rb

```diff
@@ -186,8 +186,8 @@ def sanitized_sidebar(text)
   def follow_button(followable, style = "full", classes = "")
     return if followable == Users::DeletedUser
 
-    user_follow = followable.instance_of?(User) ? "follow-user" : ""
     followable_type = followable.class_name
+    user_follow = followable_type.include?("User") ? "follow-user" : "" # User or UserDecorator
```

Comment:
I suppose this will become a problem if we ever have followable objects with "User" anywhere in the class name, but for now ... it seems fine?